src/resources/event.rs

```diff
@@ -233,6 +233,7 @@ pub enum EventObject {
     Order(Order),
     OrderReturn(OrderReturn),
     PaymentIntent(PaymentIntent),
+    PaymentMethod(Card),
```

Comment:
Why `Card`? Shouldn't this be `PaymentMethod(PaymentMethod)`?

Reply:
That's what I thought, but that's not what Stripe is returning. Just finished testing and we're definitely getting a card-id and card fields.